integration-test/shadow/CMakeLists.txt

```diff
@@ -66,3 +68,28 @@ create_test(${stest_name}
             "${test_include_directories}"
         )
 
+# Set preprocessor defines for test if configured in build.
+if(BROKER_ENDPOINT)
+    target_compile_definitions(
+        ${stest_name} PRIVATE
+            AWS_IOT_ENDPOINT="${BROKER_ENDPOINT}"
+    )
+endif()
+if(ROOT_CA_CERT_PATH)
+    target_compile_definitions(
+        ${stest_name} PRIVATE
+            ROOT_CA_CERT_PATH="${ROOT_CA_CERT_PATH}"
+    )
+endif()
+if(CLIENT_CERT_PATH)
+    target_compile_definitions(
+        ${stest_name} PRIVATE
+        CLIENT_CERT_PATH="${CLIENT_CERT_PATH}"
+    )
+endif()
+if(CLIENT_PRIVATE_KEY_PATH)
+    target_compile_definitions(
+        ${stest_name} PRIVATE
+        CLIENT_PRIVATE_KEY_PATH="${CLIENT_PRIVATE_KEY_PATH}"
+    )
+endif()
```

Comment:
Nit:please add a newline to the end of file.

Reply:
thanks